hs_core/views/__init__.py

```diff
@@ -619,6 +619,17 @@ def delete_metadata_element(request, shortkey, element_name, element_id, *args,
     return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
 
+def delete_author(request, shortkey, element_id, *args, **kwargs):
+    try:
+        res, _, _ = authorize(request, shortkey, needed_permission=ACTION_TO_AUTHORIZE.EDIT_RESOURCE)
+        res.metadata.delete_element('creator', element_id)
```

Comment:
delete_element will raise ValidationError if the creator element does not exist... I don't think there is any reason for this to be wrapped in the try/catch block.

Reply:
only wrap it if you don't like the message passed up and want to add/modify it.  Be specific about what you want to catch.

Reply:
@sblack-usu - In any case, a JSON response needs to be returned with `'status': 'error'` since this is expected by the frontend logic. Don't we need this fallback or is ValidationError the only exception that will be raised by this method?